phpBB/phpbb/controller/resolver.php

```diff
@@ -116,9 +116,18 @@ public function getController(Request $request)
 	*/
 	public function getArguments(Request $request, $controller)
 	{
-		// At this point, $controller contains the object and method name
-		list($object, $method) = $controller;
-		$mirror = new \ReflectionMethod($object, $method);
+		// At this point, $controller should be a callable
+		if (is_array($controller))
+		{
+			list($object, $method) = $controller;
+			$mirror = new \ReflectionMethod($object, $method);
+		} else if (is_object($controller) && !$controller instanceof \Closure)
```

Comment:
In accordance with the phpBB coding guidelines (read: https://area51.phpbb.com/docs/31x/coding-guidelines.html#codelayout), to keep things consistent, brackets should go on their own line. Please change this one and the two around the `else` on line 128.